modules/claims_api/app/controllers/claims_api/base_form_controller.rb

```diff
@@ -76,5 +80,14 @@ def format_526_errors(errors)
         { status: 422, detail: "#{error['key']} #{error['detail']}", source: error['key'] }
       end
     end
+
+    def track_526_validation_errors(errors)
+      StatsD.increment STATSD_VALIDATION_FAIL_KEY
+
+      errors.each do |error|
+        key = error['key'].gsub(/\[(.*?)\]/, '')
```

Comment:
Are we confident that brackets are the only thing that aren't in this whitelist? https://docs.datadoghq.com/tagging/#defining-tags

Reply:
It's actually less about the whitelist and more about making the tags less unique. 

The errors for array type objects include the index where the error occurred, this is to get rid of that index.